src/pile.js

```diff
@@ -694,6 +694,9 @@ const createPile = ({ initialItem, render, id, pubSub, store }) => {
 
     graphics.interactive = true;
     graphics.buttonMode = true;
+
+    initialItem.moveTo(0, 0);
```

Comment:
I don't think we need this. If I am not mistaken `[0,0]` is the default position anyway